project_task_type/README.rst

```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Project Task Type
+=================
+
+This module extends the functionality of Project and allow you to add a project columns to task and make project required in task and issue,and move the priority from page to form.
```

Comment:
This module extends the functionality of Project and adds the concept of task type in views and reports.